pkg/codegen/internal/test/testdata/external-resource-schema/dotnet/Workload.cs

```diff
@@ -0,0 +1,66 @@
+// *** WARNING: this file was generated by test. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading.Tasks;
+using Pulumi.Serialization;
+
+namespace Pulumi.Example
+{
+    public partial class Workload : Pulumi.CustomResource
+    {
+        [Output("pod")]
+        public Output<Pulumi.Kubernetes.Types.Outputs.Core.V1.Pod?> Pod { get; private set; } = null!;
```

Comment:
This was the tricky part: I needed to bring a map of C# namespaces for `Core.V1` and compact settings to make it `Pulumi.Kubernetes.Types.Outputs.Core.V1` instead of `Pulumi.Kubernetes.Core.V1.Outputs`.